src/aqua_diagnostics/seaice/plot_seaice.py

```diff
@@ -64,6 +65,8 @@ def __init__(self, monthly_models=None, annual_models=None,
             loglevel (str, optional): 
                 Logging level for debugging and information messages. Defaults to 'WARNING'.
         """
+        ConfigStyle(style='aqua', loglevel=loglevel)
```

Comment:
This hardcodes the style, while in genral we would like the style to be defined from the aqua config file

Reply:
also, I am not so sure that it is good to have this in the class init...maybe better in `plot_seaice`

Reply:
just call without the style keyword and it will default to the style of the config file